### PR TITLE
Hubs page edit

### PR DESCRIPTION
**Pull request by nyashamaphosa**
## Status

* Current Status: Ready for review
* Review App link(s): *populate this with links to the relevant parts of the review app*
* Closes https://github.com/NCCE/teachcomputing.org-issues/issues/2391

## What's changed?

removes bold aside heading and rearranges aside component layout

### CODE REVIEW

File: app/views/hubs/index.html.erb [hunk 1]
```diff
@@ -19,25 +19,25 @@
...
-          text: t('.sme_section_text_html'),
-          use_button: true,
+          title: t('.kit_aside_title'),
+          text: t('.kit_aside_text'),
           link: {
-            text: t('.sme_section_link_text'),
-            url: t('.sme_section_link_url'),
-            tracking_label: 'SME contact'
+            text: t('.kit_aside_link_text'),
+            url: 'https://static.teachcomputing.org/Guide-for-physical-computing-kits.pdf',
```
Comment (barrywoolgar) [inline app/views/hubs/index.html.erb:39]: You could take this opportunity to move the URL to the locale file, for consistency with the SME section and a similar change you made a few weeks ago.

File: spec/views/pages/gcse-revision_spec.rb [hunk 2]
```diff
@@ -18,6 +18,6 @@
   end
 
   it 'has three headings' do
-    expect(rendered).to have_css('.govuk-heading-m', count: 4)
+    expect(rendered).to have_css('.govuk-heading-s', count: 4)
```
Comment (barrywoolgar) [inline spec/views/pages/gcse-revision_spec.rb:21]: I love that this test is called "has three headings" and checks for four!

**Review by barrywoolgar — CHANGES_REQUESTED** [review]
Nothing broken, just a few minor improvement suggestions

File: app/components/aside_component/aside_component.html.erb [hunk 3]
```diff
@@ -11,7 +11,7 @@
     </div>
   <% end %>
   <% if @title.present? %>
-    <h2 class="<%= @use_button? 'govuk-heading-m' : 'govuk-heading-s' %> aside-component__heading">
+    <h2 class="<%= @use_button? 'govuk-heading-s' : 'govuk-heading-s' %> aside-component__heading">
```
[the remark this answers is not in the document]
Comment (nyashamaphosa) [reply app/components/aside_component/aside_component.html.erb]: resolved thank you

File: app/views/hubs/index.html.erb (hunk 1 above)
Comment (nyashamaphosa) [reply app/views/hubs/index.html.erb]: resolved too thank you

File: app/components/aside_component/aside_component.html.erb [hunk 4]
```diff
@@ -11,7 +11,7 @@
     </div>
   <% end %>
   <% if @title.present? %>
-    <h2 class="<%= @use_button? 'govuk-heading-m' : 'govuk-heading-s' %> aside-component__heading">
+    <h2 class="<%= 'govuk-heading-s' %> aside-component__heading">
```
Comment (barrywoolgar) [inline app/components/aside_component/aside_component.html.erb:14]: This doesn't need to be interpolated, you can just write it as a bare class name now.

**Review by barrywoolgar — COMMENTED** [review]
Sorry for the delay, I've been off for a few days. Just one thing to tidy up and then I can approve the PR. Thanks for handling the other items.

File: app/components/aside_component/aside_component.html.erb (hunk 4 above)
Comment (nyashamaphosa) [reply app/components/aside_component/aside_component.html.erb]: Resolved thank you. I overlooked that one somehow

**Pull request merged**